app/pages/ServiceDiscoveryResults/SearchResults/SearchResults.scss

```diff
@@ -69,4 +104,15 @@
     white-space: nowrap;
     overflow: scroll;
   }
-}
\ No newline at end of file
+
+  @media screen and (max-width: 1120px) {
```

Comment:
Could you move this and the following media query higher up? Since they affect the `.sideLinks` selector, it's easier to read if you don't have the nested selectors in between, since these media queries affect the same element as the stuff at the top, not the nested selectors.